src/Extensions/Hosting.Services.Web/Middlewares/UserIdentity/SaltProviders/RotatingSaltProvider.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
+{
+	internal class RotatingSaltProvider : ISaltProvider
```

Comment:
You could add a comment to make it clear that this implementation will generate different salts (and hashes) on different VMs

Reply:
Added comment, thanks